blocks/api/registration.js

```diff
@@ -32,6 +32,7 @@ export function registerBlockType( slug, settings ) {
 		return;
 	}
 	if ( ! /^[a-z0-9-]+\/[a-z0-9-]+$/.test( slug ) ) {
+		console.log( slug );
```

Comment:
Hmm, [it is a rule](https://github.com/WordPress/gutenberg/blob/73bd4ab0659897c9f710eaf2d888dce635e72bf4/.eslintrc.json#L53), and `npm run test` should fail on it.